integrationTests/CreateRegularGivingMandateTest.php

```diff
@@ -29,8 +31,15 @@ public function testItCreatesRegularGivingMandate(): void
         $stripeProphecy->createPaymentIntent(
             Argument::that(fn(array $payload) => ($payload['amount'] === $pencePerMonth))
         )
-            ->shouldBeCalledTimes(3)
+            ->shouldBeCalledTimes(1)
```

Comment:
```suggestion
            ->shouldBeCalledOnce()
```
Marginally clearer/ more searchable maybe? Similar in one other place below.

Reply:
Thanks, done in 59de6847fe9